File: ClickServerWeb/WebContent/com/pku/ss/egl/prog/TestKeys.js
define(["com/pku/ss/egl/libs/HotKeysControllerLib"],function(){
	egl.defineClass('com.pku.ss.egl.prog', "TestKeys", ProgramBase, {
		"eze$$fileName" : "com/pku/ss/egl/prog/TestKeys.egl",
			,
			"eze$$setEmpty": function() {
				new egl.com.pku.ss.egl.libs.HotKeysControllerLib();
			}
			,
			"eze$$setInitial": function() {
				this.eze$$setEmpty();
			}
			,
			"eze$$clone": function() {
				var ezert$$1 = this;
				var ezert$$2 = new egl.com.pku.ss.egl.prog.TestKeys();
				return ezert$$2;
			}
			,
			"eze$$getAnnotations": function() {
				if(this.annotations === undefined){
					this.annotations = {};
					this.annotations["XMLRootElement"] = new egl.eglx.xml.binding.annotation.XMLRootElement("TestKeys", null, false);
				}
				return this.annotations;
			}
			,
			"eze$$getFieldInfos": function() {
				if(this.fieldInfos === undefined){
					var annotations;
					this.fieldInfos = new Array();
				}
				return this.fieldInfos;
			}
			,
			"main": function() {
				egl.com.pku.ss.egl.libs.HotKeysControllerLib['$inst'].left();
				egl.com.pku.ss.egl.libs.HotKeysControllerLib['$inst'].backwardCmd();
			}
			,
			"toString": function() {
				return "[TestKeys]";
			}
		}
	);
});

File: ClickServer/generatedJava/com/pku/ss/egl/prog/TestKeys.java
package com.pku.ss.egl.prog;
import org.eclipse.edt.javart.resources.*;
import org.eclipse.edt.javart.*;
import com.pku.ss.egl.libs.HotKeysControllerLib;
import org.eclipse.edt.runtime.java.eglx.lang.EAny;
import java.lang.Object;
@SuppressWarnings("unused")
@javax.xml.bind.annotation.XmlRootElement(name="TestKeys")
public class TestKeys extends ProgramBase {
	private static final long serialVersionUID = 10L;
	public HotKeysControllerLib eze_Lib_com_pku_ss_egl_libs_HotKeysControllerLib;
	
	public static void main(String... ezeargs) throws Exception {
		StartupInfo info = new StartupInfo( "TestKeys", "com/pku/ss/egl/prog/TestKeys.properties", ezeargs );
		RunUnit ru = new JSERunUnit( info );
		org.eclipse.edt.javart.Runtime.setStaticRunUnit( ru );
		ru.start( new TestKeys() );
		ru.exit();
	}
	public TestKeys() {
		super();
	}
	{
		if(org.eclipse.edt.javart.Runtime.getRunUnit().getActiveExecutable() == null)
		org.eclipse.edt.javart.Runtime.getRunUnit().setActiveExecutable(this);
		ezeInitialize();
	}
	public void ezeInitialize() {
	}
	public HotKeysControllerLib eze_Lib_com_pku_ss_egl_libs_HotKeysControllerLib() {
		if (eze_Lib_com_pku_ss_egl_libs_HotKeysControllerLib == null) {
			eze_Lib_com_pku_ss_egl_libs_HotKeysControllerLib = (HotKeysControllerLib)org.eclipse.edt.javart.Runtime.getRunUnit().loadLibrary("com.pku.ss.egl.libs.HotKeysControllerLib");
		}
		return eze_Lib_com_pku_ss_egl_libs_HotKeysControllerLib;
	}
	public void main() {
		eze_Lib_com_pku_ss_egl_libs_HotKeysControllerLib().left();
		eze_Lib_com_pku_ss_egl_libs_HotKeysControllerLib().backwardCmd();
	}
}
